DD/wwwroot/Styles/ContentCarousel.css

```diff
@@ -6,42 +6,31 @@
     overflow: auto;
     color: white;
 }
-
 .text-ellipsis {
     display: inline-block;
     width: 100%;
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
 }
-
 .carousel-item {
-    min-height: 340px;
-    max-height: 360px;
+    min-height: 500px;
     width: 100%;
     position: relative;
     overflow: hidden;
 }
-.carousel-item.news-carousel{
-    min-height: 260px;
-    max-height: 320px;
-    width: 100%;
-    position: relative;
-    overflow: hidden;
+.carousel-item img {
+    max-height: 500px;
+    object-fit: cover;
 }
-
 .carousel-item-content.img-carousel img {
     width: 90% !important;
     position: relative;
-    left: 7%;
+    left: 5%;
 }
-
-.carousel-img-container.not-footer{
+.carousel-img-container {
     max-width: 500px;
-    max-height: 250px;
-    overflow: hidden;
```

Comment:
this may cause issues due to the fact that deleting the .not-footer class and just leaving it as is now also affects the footer